OpenEdXMobile/src/main/java/org/edx/mobile/deeplink/Screen.java

```diff
@@ -7,6 +7,7 @@
 public class Screen {
     public static final String COURSE_DASHBOARD = "course_dashboard";
     public static final String COURSE_VIDEOS = "course_videos";
+    public static final String COURSE_DISCUSSION = "course_discussion";
```

Comment:
As per [deep linking wiki](https://openedx.atlassian.net/wiki/spaces/LEARNER/pages/855113971/Deep+Linking) screen name should be `discussion`, or deep linking document needs to update.

Reply:
Doc has been updated.